std_msgs/Header header
string arm_type
string status
float64[6] joint_angles
float64[6] end_effector_pose
float64[6] last_goal
bool gripper_engaged
